CHANGELOG_iOS.md

```diff
@@ -1,5 +1,28 @@
 # Changelog
 
+## [1.66.113](https://github.com/brave/brave-browser/releases/tag/v1.66.113)
+
+### Web3
+
+ - Added support to retry, cancel and speed up transactions. ([#36627](https://github.com/brave/brave-browser/issues/36627))
+ - Fixed toggle being missing when importing an existing Wallet. ([#38123](https://github.com/brave/brave-browser/issues/38123))
+ - Fixed not being able to add/edit a custom network with chain ID below 16. ([#37863](https://github.com/brave/brave-browser/issues/37863))
+ - Fixed user visible assets to be only shown in Account token list. ([#36806](https://github.com/brave/brave-browser/issues/36806))
+
+### General
+
+ - Implemented using cached ad-blocking results after changing Shield settings. ([#37090](https://github.com/brave/brave-browser/issues/37090))
```

Comment:
nit: Only use `Shields` as a capital when also mentioning `Brave`. For example: `Brave Shields`. But we usually just use it as a lower case. For example:

```
 - Implemented using cached ad-blocking results after changing shields settings. ([#37090](https://github.com/brave/brave-browser/issues/37090))
```